operational-documentation/runbook.json

```diff
@@ -0,0 +1,31 @@
+{
+	"dataItemID": "origami-registry-ui",
+	"name": "Origami Registry UI",
+	"description": "Get information about Origami components, services, and repositories",
+	"serviceTier": "Bronze",
+	"lifecycleStage": "Requirements",
+	"primaryURL": "https://origami-registry-ui.ft.com",
```

Comment:
Have we thought about having it under `.in.ft.com` instead of `.ft.com`?

Reply:
Nope, but no reason why we couldn't. We should review before we go live, but `https://origami-registry.ft.com` is already set up.

@gvonkoss: there shouldn't be the `-ui` part in this URL